model-optimizer/mo/utils/versions_checker.py

```diff
@@ -127,6 +146,12 @@ def get_module_version_list_from_file(file_name, env_setup):
     req_dict = list()
     with open(file_name) as f:
         for line in f:
+            if '#' in line:
+                # Handle comments
+                pos = line.find('#')
+                line = line[:pos]
+                if line == '':
+                    continue
```

Comment:
I think you can split a line by `#` and choose the first element of splitting. And it needs just one line.

Reply:
Yes, it is a simpler solution. Updated.